lib/ship_fosdick/downloader.rb

```diff
@@ -1,27 +1,21 @@
 module ShipFosdick
   class Downloader
-    def initialize(bucket)
-      @bucket = bucket
-    end
 
-    def download
+    def self.download
       [].tap do |returned_content|
         objects.each do |object|
           next unless object.key.downcase.include?('ship')
-          returned_content << object.content
+          next unless object.key.downcase.include?('.txt')
```

Comment:
Also, I don't know how likely it is, but if you can have a `.txt.zip` for instance or else, maybe you'd want the rule to be more precise with an `end_with?` instead of `include?`. 
Maybe ?! Maybe not !